lit_gpt/model.py

```diff
@@ -87,6 +87,9 @@ def forward(self, idx: torch.Tensor, input_pos: Optional[torch.Tensor] = None) -
             mask = None
 
         x = self.transformer.wte(idx)  # token embeddings of shape (b, t, n_embd)
+        # NOTE: this is a secret sauce (Gemma)
```

Comment:
good catch: https://github.com/google/gemma_pytorch/blob/ca890c7abaa41ce7ab0eeda9aa8a52c0796b3a16/gemma/model.py#L431C1-L432C1

Reply:
Definitely a good catch. I don't think that was in the paper or I totally glanced over this.